imporve the export speed

jianwan0214 opened the pull request:
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

fengttt commented:
Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?

daviszhen commented on pkg/frontend/export.go:
```diff
@@ -37,15 +38,16 @@ func (cld *CloseExportData) Close() {
 
 func initExportFileParam(ep *tree.ExportParam, mrs *MysqlResultSet) {
 	ep.DefaultBufSize *= 1024 * 1024
+	fmt.Println("wangjian sql2 is", ep.DefaultBufSize)
```
remove it

jianwan0214 commented:
> Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?

yes, I changed the string to []byte to avoid some memory copy expenses as convert string to []byte. Here is the ten million rows data test result.

<img width="945" alt="WX20220505-144146@2x" src="https://user-images.githubusercontent.com/32733096/166874106-2fda0867-f528-4774-b091-3480ff40a3b8.png">

jianwan0214 commented:
> > Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?
> 
> yes, I changed the string to []byte to avoid some memory copy expenses as convert string to []byte. Here is the ten million rows data test result.
> 
> <img alt="WX20220505-144146@2x" width="945" src="https://user-images.githubusercontent.com/32733096/166874106-2fda0867-f528-4774-b091-3480ff40a3b8.png">

the first is the origin version. the second is the new version.

fengttt commented:
> > > Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?
> > 
> > 
> > yes, I changed the string to []byte to avoid some memory copy expenses as convert string to []byte. Here is the ten million rows data test result.
> > <img alt="WX20220505-144146@2x" width="945" src="https://user-images.githubusercontent.com/32733096/166874106-2fda0867-f528-4774-b091-3480ff40a3b8.png">
> 
> the first is the origin version. the second is the new version.

You got me really really worried -- is your example saying that, original version take 27 seconds, and the new code take 18 seconds to *CRASH*?

jianwan0214 commented:
> 
no, the second test is caused by restarting the mo-server, not by my code. I need to use two version code, so I need to rebuild the code and restart the mo-server to use the function. I use this just try to compare the two version code. Here is the correct operayion.
![1651739870663](https://user-images.githubusercontent.com/32733096/166888916-7d30d9de-dab5-45ee-a158-108d308e3c00.jpg)
